packages/core/src/App/Components/Layout/Header/platform-switcher.jsx

```diff
@@ -47,10 +47,12 @@ class PlatformSwitcher extends React.PureComponent {
                 >
                     <Icon
                         className='platform-switcher__icon'
-                        icon={getPlatformIcon(this.props.app_routing_history)}
+                        icon={getPlatformInformation(this.props.app_routing_history).icon}
                         size={32}
                     />
-                    <h1 className='platform-switcher__header'>{getPlatformHeader(this.props.app_routing_history)}</h1>
+                    <h1 className='platform-switcher__header'>
+                        {getPlatformInformation(this.props.app_routing_history).header}
```

Comment:
Same here

Reply:
yes, that's a fixed property which is always available.